generator/C/test/posix/Makefile

```diff
@@ -1,4 +1,4 @@
-CFLAGS = -g -Wall -Werror -O0
+CFLAGS = -g -Wall -Werror -O0 -Wno-address-of-packed-member
```

Comment:
Please add a comment here to explain why it's OK to add this compiler directive in the case of mavlink bindings.

Reply:
without this, i get many-many (166) build failures from the mavlink library.... and with it the problem goes away....
example error:
../../include_v2.0/ardupilotmega/../common/testsuite.h: In function ‘mavlink_test_distance_sensor’:
../../include_v2.0/ardupilotmega/../common/testsuite.h:6417:292: error: taking address of packed member of ‘struct __mavlink_distance_sensor_t’ may result in an unaligned pointer value [-Werror=address-of-packed-member]

number of errors without the fix:
buzz@buzz:~/GCS/mavlink/pymavlink/generator/C/test/posix$ make clean testmav1.0_ardupilotmega testmav2.0_ardupilotmega testmav1.0_common testmav2.0_common 2>&1 | grep 'error:' | wc
    166    4770   60806

number of errors with the fix:
buzz@buzz:~/GCS/mavlink/pymavlink/generator/C/test/posix$ make clean testmav1.0_ardupilotmega testmav2.0_ardupilotmega testmav1.0_common testmav2.0_common 2>&1 | grep 'error:' | wc
      0       0       0